FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2DrawerActivity.kt

```diff
@@ -64,7 +41,8 @@ class V2DrawerActivity : V2DemoActivity() {
     }
 
     override val paramsUrl = "https://github.com/microsoft/fluentui-android/wiki/Controls#params-17"
-    override val controlTokensUrl = "https://github.com/microsoft/fluentui-android/wiki/Controls#control-tokens-17"
+    override val controlTokensUrl =
```

Comment:
remove additional space

Reply:
Formatting file change. not added intentionally.